File: platforms/python/examples/core/pass/add_mesh_to_shader.py
from fragmentcolor import Shader, Mesh, Vertex, Pass

mesh = Mesh()
mesh.add_vertex([0.0, 0.0])
shader = Shader.from_mesh(mesh)
rpass = Pass("rpass"); rpass.add_shader(shader)

rpass.add_mesh_to_shader(mesh, shader)

File: platforms/web/examples/core/pass/add_mesh_to_shader.js
import { Shader, Mesh, Vertex, Pass } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertex([0.0, 0.0]);
const shader = Shader.fromMesh(mesh);
const pass = new Pass("pass"); pass.addShader(shader);

pass.addMeshToShader(mesh, shader);
